Python/bltinmodule.c

```diff
@@ -578,12 +578,13 @@ format as builtin_format
 
 Return value.__format__(format_spec)
 
-format_spec defaults to the empty string
+format_spec defaults to the empty string.
+See help('FORMATTING') for description.
```

Comment:
The initial section of help('FORMATTNG') describes replacement fields and everything but format specs.  This could confuse a beginner.  So I suggest as an alternative:
See the Format Specification Mini-Language section of help('FORMATTING').